library/iFixit/Matryoshka/APCu.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace iFixit\Matryoshka;
+
+use iFixit\Matryoshka;
+
+class APCu extends Backend {
+   public static function isAvailable() {
+      return function_exists('apcu_enabled') && apcu_enabled();
+   }
+
+   public function set($key, $value, $expiration = 0) {
+      return apcu_store($key, $value, $expiration);
+   }
+
+   public function setMultiple(array $values, $expiration = 0) {
+      return apcu_store($values, null, $expiration);
+   }
+
+   public function add($key, $value, $expiration = 0) {
+      return apcu_add($key, $value, $expiration);
+   }
+
+   public function increment($key, $amount = 1, $expiration = 0) {
+      $value = apcu_inc($key, $amount, $success);
+
+      // Call set() if the key doesn't exist.
+      if ($success) {
+         return $value;
+      } else if ($this->set($key, $amount, $expiration) !== false) {
+         return $amount;
+      } else {
+         return false;
+      }
+   }
+
+   public function decrement($key, $amount = 1, $expiration = 0) {
+      return $this->increment($key, -$amount, $expiration);
+   }
+
+   public function get($key) {
+      $value = apcu_fetch($key, $success);
+
+      return $success ? $value : self::MISS;
+   }
+
+   public function getMultiple(array $keys) {
+      if (empty($keys)) {
+         return [[],[]];
+      }
+
+      // Default to an empty array in case no keys were found.
+      $hits = apcu_fetch(array_keys($keys)) ?: [];
+
+      $found = [];
+      $missed = [];
+      foreach ($keys as $key => $id) {
+         $value = array_key_exists($key, $hits) ? $hits[$key] : self::MISS;
+         $found[$key] = $value;
+
+         if ($value === self::MISS) {
+            $missed[$key] = $id;
```

Comment:
Good catch, but that's how the function is described:
https://github.com/iFixit/Matryoshka/blob/master/library/iFixit/Matryoshka/Backend.php#L71-L88